data/json/monsterdrops/feral_humans.json

```diff
@@ -668,34 +668,19 @@
     "magazine": 100,
     "ammo": 0,
     "entries": [
-      { "distribution": [ { "group": "clothing_soldier_broken_armor_set", "prob": 100, "damage": [ 2, 4 ] } ] },
+      {
+        "distribution": [
+          { "group": "clothing_soldier_set", "prob": 65, "damage": [ 1, 3 ] },
```

Comment:
I would think ferals could even have a chance of 0 damage.

Reply:
Aight, that would have to be 0, though, because having one there is what caused the initial issue

Reply:
Aight, that would have to be 0, though, because having one there is what caused the initial issue